src/modules/fw_pos_control_l1/fw_pos_control_l1_params.c

```diff
@@ -1027,3 +974,59 @@ PARAM_DEFINE_FLOAT(FW_WING_SPAN, 6.0);
  * @group FW Geometry
  */
 PARAM_DEFINE_FLOAT(FW_WING_HEIGHT, 0.5);
+
+/**
+ * Landing flare time
+ *
+ * Multiplied by the descent rate to calculate a dynamic altitude at which
```

Comment:
I would add a comment here that only the max() of FW_LND_FLALT and FW_LND_FL_TIME* vz is taken.

Reply:
done [dec33ca](https://github.com/PX4/PX4-Autopilot/pull/19871/commits/dec33ca69ea7f98093d0c3880343330849d58560)